### PR TITLE
ci: unit tests

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

closes: #13

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: app/test_helpers.go [hunk 1]
```diff
@@ -273,7 +273,6 @@ func IntegrationTestNetworkConfig() network.Config {
 	cfg.LegacyAmino = encCfg.Amino
 	cfg.InterfaceRegistry = encCfg.InterfaceRegistry
 	cfg.GenesisState = appGenState
-	cfg.BondDenom = params.BondDenom
```
Comment (adamewozniak) [inline app/test_helpers.go:276]: This is an interesting bit that's in [umee's helpers](https://github.com/umee-network/umee/blob/36171b24fbb225e2df1b1116348b9e56d3911b57/app/test_helpers.go#L316), but breaks on the most recent version of the SDK. 

This is likely not an issue there bc Umee is on an SDK fork

**Comment by adamewozniak** [comment]
ok so we're failing on this:

```
11:43PM ERR CONSENSUS FAILURE!!! err="parameter MaxValidators not registered" module=consensus stack="goroutine 121 [running]:\nruntime/debug.Stack()\n\t/opt/hostedtoolcache/go/1.19.2/x64/src/runtime/debug/stack.go:24 +0x65\ngithub.com/tendermint/tendermint/consensus.
```

**Comment by adamewozniak** [comment]
okay well, reproducing locally is not doing so well

File: app/test_helpers.go (hunk 1 above)
Comment (adamewozniak) [reply app/test_helpers.go:276]: jk! this was an issue because I forgot to override the default genesis for a few modules. got it now

**Comment by adamewozniak** [comment]
woohoo! bloxs!

This was due to the cacheing of the binary in the [liveness test workflow](cd6c4513bd3c7a6a942c1b2f4e5d3775cb832d00) .

File: x/oracle/types/msgs_test.go [hunk 2]
```diff
@@ -94,9 +94,9 @@ func TestMsgAggregateExchangeRateVote(t *testing.T) {
 	msgInvalidUnknownRequest := "must provide at least one oracle exchange rate: unknown request"
 	msgInvalidFeederAddr := "invalid feeder address (empty address string is not allowed): invalid address"
 	msgInvalidOperatorAddr := "invalid operator address (empty address string is not allowed): invalid address"
-	msgInvalidOraclePrice := "failed to parse exchange rates string cause: invalid oracle price: invalid coins"
+	msgInvalidOraclePrice := "oracle price"
```
Comment (zarazan) [inline x/oracle/types/msgs_test.go:97]: These error messages seem a lot less descriptive. I don't see where these changed in the code, was it always this way?

Comment (adamewozniak) [reply x/oracle/types/msgs_test.go:97]: The error messages in the module are actually more descriptive now; there are just multiple different cases
But, they all include this text

**Review by zarazan — APPROVED** [review]
Looks good. That single-node script is a little gnarly but has some good comments.

**Pull request merged**